pom.xml

```diff
@@ -49,7 +49,33 @@
                 <artifactId>target-platform-configuration</artifactId>
                 <version>${tycho.version}</version>
                 <configuration>
-                    <resolver>p2</resolver>
+                  <environments>
```

Comment:
why do we need these `environments` configurations?
it gives me a feeling that TestNG plugin will build env specific bundles, but it does not.